src/connection.ts

```diff
@@ -46,14 +45,13 @@ type ConnectionTestHarness = {
   protocolVersion?: string | undefined;
 };
 
-const { packageJson } = readPackageUp.sync({ cwd: __dirname }) || {};
 const API_URL =
   process.env["WHEROBOTS_API_URL"] || "https://api.cloud.wherobots.com";
 
 const PROTOCOL_VERSION = "1.0.0";
 const OS_TYPE = `${process.platform};${process.arch}`;
 const NODE_VERSION = process.version;
-const USER_AGENT = `${packageJson?.name}/${packageJson?.version} os/${OS_TYPE} node/${NODE_VERSION}"`;
```

Comment:
i'd rather not lose the package version from the user agent string. is there a way we can inject this as a build-time variable?